06.bowling_object/lib/game.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require_relative 'frame'
+
+class Game
+  attr_reader :frames
+
+  def initialize(shot_pins)
+    @frames = build_frames(shot_pins)
+  end
+
+  def score
+    @frames.each_with_index.sum { |frame, idx| frame.score(@frames, idx) }
```

Comment:
```suggestion
    @frames.each_with_index.sum { |frame| frame.score(@frames) }
```

Frameをnewしたタイミングでフレーム番号を覚えさせて、Frame自身が自分は何番目のフレームかわかっているようにした方がベターかなと思います。

Reply:
Frame に attr_reader :number を追加しました。